Duplicati/Server/Program.cs

```diff
@@ -380,6 +380,8 @@ public static void RealMain(string[] args)
                 if (commandlineOptions.ContainsKey("webservice-password"))
                     Program.DataConnection.ApplicationSettings.SetWebserverPassword(commandlineOptions["webservice-password"]);
 
+                Program.DataConnection.ApplicationSettings.GenerateWebserverPasswordTrayIcon();

```

Comment:
This would mean that the TrayIcon login feature is enabled for all users.
Can the TrayIcon generate the password instead? That way it will only be generated for users who want this feature.
We then need to make sure the login does not accept an empty password, so we do not open all machines.